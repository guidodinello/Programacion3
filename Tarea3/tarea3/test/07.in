leer_dias 12
leer_precios 2 7 6 1 8 10 5 11 8 19 5 40
leer_disparos 2
k_disparos
leer_disparos 3
k_disparos
leer_disparos 4
k_disparos
leer_disparos 5
k_disparos
Fin