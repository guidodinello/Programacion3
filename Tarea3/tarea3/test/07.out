1>2>3>4>Con precios: 2 7 6 1 8 10 5 11 8 19 5 40 
La ganancia maxima con 2 disparo(s) es: 53000 
5>6>Con precios: 2 7 6 1 8 10 5 11 8 19 5 40 
La ganancia maxima con 3 disparo(s) es: 58000 
7>8>Con precios: 2 7 6 1 8 10 5 11 8 19 5 40 
La ganancia maxima con 4 disparo(s) es: 63000 
9>10>Con precios: 2 7 6 1 8 10 5 11 8 19 5 40 
La ganancia maxima con 5 disparo(s) es: 66000 
11>Fin.
